Source/ACE/Command/Handlers/AdminCommands.cs

```diff
@@ -1193,24 +1204,35 @@ public static void HandleHousekeep(Session session, params string[] parameters)
         }
 
         // idlist
-        [CommandHandler("idlist", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld, 0)]
+        [CommandHandler("idlist", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld, 0,
+            "Shows the next ID that will be allocated from SQL.")]
         public static void HandleIDlist(Session session, params string[] parameters)
```

Comment:
you're reading the function wrong. It's not the current player's guid. The +1 is wrong as I didn't realize current was already set to +1 when Alloc is called. I've fixed the +1 issue